{"uuid": "6df6c4ed-f55c-46a7-9fcd-05501c8096d0", "children": ["f8406b49-2fca-4e12-a0d0-c86a71e77976"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "NameError: name 'EC' is not defined\n", "trace": "  File \"C:\\Users\\Viktoriya\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Viktoriya\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\Viktoriya\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\Viktoriya\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\Viktoriya\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\cygwin64\\home\\Viktoriya\\Practikum_repos\\Sprint_6\\tests\\conftest.py\", line 12, in driver\n    browser.quit()\n        ^^\n"}, "start": 1743949470285, "stop": 1743949512602}], "start": 1743949470285, "stop": 1743949513307}